pkg/compose/cp.go

```diff
@@ -57,23 +57,32 @@ func (s *composeService) Copy(ctx context.Context, projectName string, options a
 		if options.All {
 			return errors.New("cannot use the --all flag when copying from a service")
 		}
+		// due to remove of the --all flag, restore the default value to 1 when copying from a service container to the host.
+		if options.Index == 0 {
+			options.Index = 1
```

Comment:
I wonder: when user run `docker compose cp service:/path /tmp` (copy from container to local) without index, shall we force index=1, or just use first indexed container (which might not be `1`) ? This can be addressed in a follow-up PR, just noting

Reply:
Yes, that's what I was wondering in my comment above;

> As to the flag itself; are indexes guaranteed to start with 1? (i.e., could a container be removed, and it starts with 2 ?)

Reply:
Ok, I'll address that point in this PR